calibration/calibrate_model.sh

```diff
@@ -41,6 +41,8 @@ create_measure_config() {
 
     if [[ $model_name_lower =~ ^mixtral ]]; then
         tmp_config="{\"method\": \"HOOKS\",\"mode\": \"MEASURE\",\"observer\": \"maxabs\",\"allowlist\": {\"types\": [], \"names\":  []},\"blocklist\": {\"types\": [], \"names\":  [\"self_attn\", \"lm_head\"]},\"quantize_weight\": false,\"dump_stats_path\": \"$1/$2/$3/inc_output\"}"
+    elif [[ $model_name_lower =~ ^deepseek ]]; then
+        tmp_config="{\"method\": \"HOOKS\",\"mode\": \"MEASURE\",\"observer\": \"maxabs\",\"allowlist\": {\"types\": [], \"names\":  []},\"blocklist\": {\"types\": [], \"names\":  [\"lm_head\", \"mlp\\\.gate\\\b\", \"block2batch_matmul\"]},\"quantize_weight\": false,\"dump_stats_path\": \"$1/$2/$3/inc_output\"}"
```

Comment:
@yiliu30 , do we still need to skip block2batch_matmul?
According to my quant_config for deepseek, I didn't skip block2batch_matmul

Reply:
Not necessary, we can remove it from the `blocklist`.